cs/cs.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27428.2043
+# Visual Studio Version 16
```

Comment:
Is this intentional? Not sure if we are communicating VS version change to 2019.

Reply:
This is just some [metadata that indicates which version of VS last touched the file](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019).